R/new.R

```diff
@@ -77,23 +84,27 @@ update_tibble_attrs <- function(x, ...) {
 
 guess_nrow <- function(x) {
   if (!is.null(.row_names_info(x, 0L))) {
-    .row_names_info(x, 2L)
+    list("nrow" = .row_names_info(x, 2L), "method" = "attribute")
```

Comment:
Maybe  `"row.names attribute"`?

The quotes in `"nrow"` and `"method"` look odd.